index.d.ts

```diff
@@ -153,6 +153,117 @@ interface Request {
   }
 }
 
+interface ContentOptions {
+  /**
+   * Controls the way the HTMLRewriter treats inserted content.
+   *
+   * - true: Raw HTML
+   * - false: Text and any HTML will be escaped
+   */
+  html: boolean
```

Comment:
It's probably worth mentioning that if this options object is not provided, then the default value will be `false`